core/src/main/java/io/varhttp/VarHttpContext.java

```diff
@@ -74,7 +55,7 @@ public int read() throws IOException {
 				parsePostData.putAll(HttpHelper.parseQueryString(wwwForm));
 			}
 		} finally {
-			newInput.reset();
+//			is.reset();
```

Comment:
is it not being reset because it could be a websocket?

Reply:
This is fixed in the merge branch, see other pr